### PR TITLE
Allow Tuple{Bool, Bool} for xtrimspine and ytrimspine.

### LINKED ISSUE #2169: Support both "Bool" and "Tuple{Bool, Bool}" for "xtrimspine" and "ytrimspine"
I think the meaning would be self-explanatory. Sometimes you want to trim only one end of the spine and currently that's not possible. 

I would be happy to implement this if this is okay with the maintainers.

### PR DESCRIPTION
**Pull request by marcpabst**
# Description

Fixes #2169 

Allows to pass both a `Bool` and a `Tuple{Bool, Bool}` for `xtrimspine` and `ytrimspine`.

## Type of change

Delete options that do not apply:

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] Added an entry in NEWS.md (for new features and breaking changes)
- [x] Added or changed relevant sections in the documentation
- [ ] Added unit tests for new algorithms, conversion methods, etc.
- [ ] Added reference image tests for new plotting functions, recipes, visual options, etc.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## master
 
+- `Axis` does now accept both a `Bool` and a `Tuple{Bool, Bool}` as values for `xtrimspine` and `ytrimspine` to trim only one end of the spine.
```
Comment (asinghvi17) [inline NEWS.md:5]:
```suggestion
- `Axis` does now accept both a `Bool` and a `Tuple{Bool, Bool}` as values for `xtrimspine` and `ytrimspine` to trim only one end of the spine.  [#2171](https://github.com/JuliaPlots/Makie.jl/pull/2171)

**Comment by marcpabst** [comment]
Can someone point me to how I would implement a test for this?

**Review by jkrumbiegel — APPROVED** [review]
Looks good, thanks. For a test, have a look at how the other visual reference tests are implemented and just make a similar one where you have each combination of trims in a couple Axes in a GridLayout. Make the spine width thick so the visual differ would pick up on any changes.

**Review by SimonDanisch — APPROVED** [review]

**Comment by jkrumbiegel** [comment]
Closed in favor of https://github.com/MakieOrg/Makie.jl/pull/2279

**Pull request closed without merging**